client/karma.conf.js

```diff
@@ -20,16 +20,23 @@ webpackConfig.externals = {
   'react-addons-test-utils': 'react-dom',
 };
 
+function getSpecs(specList) {
+  if (specList) {
+    return specList.split(',');
+  } else {
+    return [
+      { pattern: 'app/**/*.spec.js', watched: true },
```

Comment:
Hmm not 100% sure why it's broken up like this, can you explain? Thanks!

Reply:
In the ```client/README.md``` I documented a way to run specific tests. I looked up on Stackoverflow on how to do this, and this was the best answer I could find so far. So if you specify a list in the environment properties, it will run those, otherwise falling back to all tests.